MarshalTests/MarshalTests.swift

```diff
@@ -15,7 +15,14 @@ import XCTest
 
 class MarshalTests: XCTestCase {
     
-    let object: MarshalDictionary = ["bigNumber": NSNumber(value: 10_000_000_000_000), "foo" : (2 as NSNumber), "str": "Hello, World!", "array" : [1,2,3,4,7], "object": ["foo" : (3 as NSNumber), "str": "Hello, World!"], "url":"http://apple.com",  "junk":"garbage", "urls":["http://apple.com", "http://github.com"]]
+    let object: MarshalDictionary = ["bigNumber": NSNumber(value: 10_000_000_000_000),
```

Comment:
I put these on multiple lines because I had to cast pretty much everything and side scrolling as annoying.